mithril.js

```diff
@@ -49,9 +49,9 @@ var m = (function app(window, undefined) {
 	 *
 	 */
 	function m(tag, pairs) {
-		for (var args = [], i = 1; i < arguments.length; i++) {
-			args[i - 1] = arguments[i];
-		}
+		// JSX compatibility with 'null' {pairs} argument
+		var args = Array.prototype.slice.call(arguments, (pairs === null ? 2 : 1));
```

Comment:
Thank you, I'll fix it.
On Sat, Aug 22, 2015 at 9:53 AM Isiah Meadows <notifications@github.com>
wrote:

> In mithril.js
> <https://github.com/lhorie/mithril.js/pull/772#discussion_r37693412>:
>
> > @@ -49,9 +49,9 @@ var m = (function app(window, undefined) {
> >  	 *
> >  	 */
> >  	function m(tag, pairs) {
> > -		for (var args = [], i = 1; i < arguments.length; i++) {
> > -			args[i - 1] = arguments[i];
> > -		}
> > +		// JSX compatibility with 'null' {pairs} argument
> > +		var args = Array.prototype.slice.call(arguments, (pairs === null ? 2 : 1));
>
> Nit: just set i to pairs === null ? 2 : 1. Otherwise, you're creating a
> perf regression.
>
> (Chrome and Firefox both don't usually like it when you pass the arguments
> object as an parameter. This is one case.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lhorie/mithril.js/pull/772/files#r37693412>.
>